assets/css/input.css

```diff
@@ -172,15 +172,24 @@ section {
 }
 
 .hello__nav__item__selected {
-    @apply  relative text-primary-yellow hover:text-primary-yellow cursor-default;
+    @apply relative text-primary-yellow hover:text-primary-yellow cursor-default;
 }
 
 .hello__nav__phone .hello__nav__item, .hello__nav__phone .hello__nav__item__yellow {
     @apply px-3 py-4;
 }
 
 .hello__nav__logo {
-    @apply md:hidden h-10 m-2;   
+    @apply hidden md:flex h-10 m-2;
+}
+
+.hello__nav__logo__filler {
+    @apply grow;
+    max-width: 40vw;
+}
+
+.hello__nav__logo__phone {
+    @apply md:hidden h-10 m-2;
 }
```

Comment:
Pokud to čtu správně spolu s [touto](https://github.com/klubFITpp/klubfitpp/pull/78/files#diff-9c4a34e464067c958759c74541311573208a3b6d4f48ed862e188d7d912c91ffR2) změnou (rename `hello__nav__logo` na `hello__nav__logo__phone`) tak mi to nedává smysl.

`hello__nav__logo` může zůstat se stejným názvem a stačí jen upravit styl na

```css
.hello__nav__logo {
   margin: 0.5rem;
   height: 2.5rem;
 }

 @media (min-width: 768px) {
   .hello__nav__logo {
     display: flex;
   }
 }
```

respektive ekvivalent s `@apply`. Takhle tu jinak vzniká mrtvý nepoužitý kód, protože pak `hello__nav__logo` už nikde neexistuje (byl přejmenovaný).

Možná se ale pletu?